tests/test_run_notebook.py

```diff
@@ -0,0 +1,29 @@
+from run_notebook import add_parameters_to_cell
+
+
+def test_add_parameters_to_cell():
+    cell_text = "Interactive(sdata=sdata, headless=True)"
+    expected_text = 'Interactive(sdata=sdata, headless=True, coordinate_system_name="global", _tested_notebook="test_notebook.ipynb", _test_target="cell5", _take_screenshot=True)'
+
+    assert add_parameters_to_cell(cell_text, take_screenshot=True) == expected_text
+
+
+def test_add_parameters_to_cell_no_interactive():
+    cell_text = "print('Hello world')"
+    expected_text = cell_text
+
+    assert add_parameters_to_cell(cell_text, take_screenshot=True) == expected_text
+
+
+def test_add_parameters_to_cell_take_screenshot_is_false():
+    cell_text = "Interactive(sdata=sdata, headless=True)"
+    expected_text = 'Interactive(sdata=sdata, headless=True, coordinate_system_name="global", _tested_notebook="test_notebook.ipynb", _test_target="cell5", _take_screenshot=False)'
+
+    assert add_parameters_to_cell(cell_text, take_screenshot=False) == expected_text
+
+
+def test_add_parameters_to_cell_screenshot_overwrite():
+    cell_text = 'Interactive(sdata=sdata, headless=True, coordinate_system_name="global", _tested_notebook="test_notebook.ipynb", _test_target="cell5", _take_screenshot=False)'
```

Comment:
We need some extra tests for the case in which the cell is formatted using blacks and so the `Interactive` parameters are split among different lines. Please add these series of tests. You can use
```
%load_ext jupyter_black
```
in the first cell of a notebook to enable it and then create dummy cells with the code of the tests (`Interactive(...)`). By running the cell, black will be execute and will reformat the code in it. You can then paste the code in the tests.